service/src/test/java/com/google/auto/service/processor/AutoServiceProcessorTest.java

```diff
@@ -46,4 +48,30 @@ public void autoService() {
             JavaFileObjects.forResource("META-INF/services/test.SomeService"),
             JavaFileObjects.forResource("META-INF/services/test.AnotherService"));
   }
-}
\ No newline at end of file
+
+  @Test
+  public void multiService() {
+    assert_().about(javaSources())
```

Comment:
`JavaSourcesSubjectFactory` has a `static` `assertThat` method that accepts a varargs.

Reply:
I'm not seeing this, can you link me where that's defined?

Reply:
Whoops, I meant `JavaSourcesSubject` (not the factory)

Reply:
94705d7